### PR TITLE
Publishing the ADR for the discussions and decisions made regarding the storage services used for files attached to Works

### LINKED ISSUE #379: Drafting an ADR documenting the team decision to store pre-curation uploads in ActiveStorage and post-curation uploads into Amazon S3 Buckets

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Resolves #379

### CODE REVIEW

**Review by bess — CHANGES_REQUESTED** [review]
Thank you for writing this! I'd like to talk about this as a team, though, because I'm not sure I understand all of it. 

In particular, I'm having trouble understanding this sentence and I wonder if we can re-word it to make it more clear:
"Advancing `Works` to the `post-curation` state will require that the S3 Bucket for `post-curation` S3 Objects be created in response to an update in the state of the `Work` object."

File: architecture-decisions/0005-work-attachment-curation-state.md
Comment (jrgriffiniii) [inline architecture-decisions/0005-work-attachment-curation-state.md:22]: A significant consequence of having `post-curation` S3 Objects managed without ActiveStorage is that ordering will *not* be preserved.

**Comment by jrgriffiniii** [comment]
This might be blocked until https://github.com/pulibrary/pdc_describe/issues/405 is resolved. I will defer to others.

**Review by bess — APPROVED** [review]

**Review by carolyncole — DISMISSED** [review]

**Pull request merged**
